appengine/gophers/gophers-5/static/index.js

```diff
@@ -27,7 +27,7 @@ window.addEventListener('load', function () {
       // User is signed in.
       document.getElementById('sign-out').hidden = false;
       document.getElementById('post-form').hidden = false;
-      document.getElementById('account-details').textContent =
+      var account = document.getElementById('account-details').textContent =
```

Comment:
We should delete `var account =` from gophers-6, rather than add it to gophers-5. It's not needed.